Formula/snctl.rb

```diff
@@ -5,21 +5,21 @@
 class Snctl < Formula
   desc "StreamNative Cloud CLI (snctl)"
   homepage "https://streamnative.io/"
-  version "0.13.3"
+  version "0.14.0"
   license "Apache-2.0"
 
   on_macos do
-    if Hardware::CPU.intel?
```

Comment:
I guess we don't have mac && intel cases anymore?

Reply:
This was published using goreleaser, and I do think that mac/intel and mac/arm are supported.  I don't know the details of the generated brew script.
https://github.com/streamnative/cloud-cli/releases/tag/v0.14.0